e2e-tests/steps/reports.ts

```diff
@@ -0,0 +1,72 @@
+import { Page, expect } from '@playwright/test'
+import Excel, { CellValue } from 'exceljs'
+
+const reportMetaData = {
+  submittedApplications: {
+    columnNames: [
+      'eventId',
+      'applicationId',
+      'personCrn',
+      'personNoms',
+      'referringPrisonCode',
+      'preferredAreas',
+      'hdcEligibilityDate',
+      'conditionalReleaseDate',
+      'submittedAt',
+      'submittedBy',
+      'createdAt',
+    ],
+    callToAction: "Download 'Submitted applications' report",
+  },
+  applicationStatusUpdates: {
+    columnNames: ['eventId', 'applicationId', 'personCrn', 'personNoms', 'updatedBy', 'updatedAt', 'newStatus'],
+    callToAction: "Download 'Application status updates' report",
+  },
+  unsubmittedApplications: {
+    columnNames: ['applicationId', 'personCrn', 'personNoms', 'startedBy', 'startedAt'],
+    callToAction: "Download 'Un-submitted applications' report",
+  },
+}
+
+type ReportType = keyof typeof reportMetaData
+
+export const signInAsMI = async (page: Page, miUser: { name?: string; username: string; password: string }) => {
+  await page.getByLabel('Username').fill(miUser.username)
+  await page.getByLabel('Password').fill(miUser.password)
+  await page.getByRole('button', { name: 'Sign in' }).click()
+}
+
+export const skipForNow = async (page: Page) => {
+  await page.getByRole('button', { name: 'Skip for now' }).click()
+  await expect(page.locator('h1')).toContainText('CAS2: Short-term accommodation')
+}
+
+export const manageInformationReports = async (page: Page) => {
+  await page.getByRole('link', { name: 'Management information reports' }).click()
+  await expect(page.locator('h1')).toContainText('Management information reports')
+}
+
+export const downloadReport = async (reportType: ReportType, page: Page) => {
+  const downloadPromise = page.waitForEvent('download')
+  await page.getByRole('button', { name: reportMetaData[reportType].callToAction }).click()
+  const download = await downloadPromise
+  const path = await download.path()
+  return path
+}
+
+export const confirmColumnNames = async (reportType: ReportType, path: string) => {
+  const workbook = new Excel.Workbook()
+
+  await workbook.xlsx.readFile(path).then(() => {
+    const sh = workbook.getWorksheet('Sheet0')
+
+    const headerCells: CellValue[] = []
+    sh.getRow(1).eachCell(cell => headerCells.push(cell.value))
+
+    expect(headerCells.length === reportMetaData[reportType].columnNames.length).toBe(true)
```

Comment:
This assertion means that if the report is updated on the API side without an update on the FE side this test will fail. I was in two minds about this scenario blocking the FE pipeline, but let me know your thoughts.